api/account/api.go

```diff
@@ -107,6 +111,9 @@ func ConvertToKCUser(user AccountRepresentation) kc.UserRepresentation {
 	if user.PhoneNumber != nil {
 		attributes["phoneNumber"] = []string{*user.PhoneNumber}
 	}
+	if user.Locale != nil {
+		attributes["locale"] = []string{*user.Locale}
```

Comment:
I cannot directly put this comment on the right line (128), I think the validation of the locale is missing in the Validate method